### ISSUE TITLE
Cli - NPE when attempting to log in using certs with new line character.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#11463]
rancher cli version - v1.0.0-alpha5

NPE when attempting to log in using certs with new line character.
```
Sangeethas-MBP:rancher-v1.0.0-alpha5 sangeethahariharan1$ ./rancher login -t token-v4nx8:cscgflc5kbtgcxj9tx9znjq7l5j9nxjrg5vftwn5dzngtq9tvnglg5 --cacert ~/ca-cert --name test1 https://138.197.107.170/v3
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
    panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x19546e2]

goroutine 1 [running]:
github.com/rancher/cli/vendor/github.com/urfave/cli.HandleAction.func1(0x42295cd30)
    /go/src/github.com/rancher/cli/vendor/github.com/urfave/cli/app.go:472 +0x2c8
panic(0x1a4dce0, 0x21e5910)
    /usr/local/go/src/runtime/panic.go:491 +0x283
github.com/rancher/cli/cmd.loadAndVerifyCert(0x7fff5fbffb67, 0x22, 0x6, 0x7fff5fbffb67, 0x22, 0x0)
    /go/src/github.com/rancher/cli/cmd/common.go:42 +0x82
github.com/rancher/cli/cmd.loginSetup(0x4227e0b40, 0x0, 0x0)
    /go/src/github.com/rancher/cli/cmd/login.go:81 +0x530
reflect.Value.call(0x1a15fc0, 0x1bce3a8, 0x13, 0x1b60900, 0x4, 0x42295ccd0, 0x1, 0x1, 0x42295cc50, 0x1b38d00, ...)
    /usr/local/go/src/reflect/value.go:434 +0x905
reflect.Value.Call(0x1a15fc0, 0x1bce3a8, 0x13, 0x42295ccd0, 0x1, 0x1, 0x4227e0b40, 0x84224ff5e8, 0x0)
    /usr/local/go/src/reflect/value.go:302 +0xa4
github.com/rancher/cli/vendor/github.com/urfave/cli.HandleAction(0x1a15fc0, 0x1bce3a8, 0x4227e0b40, 0x0, 0x0)
    /go/src/github.com/rancher/cli/vendor/github.com/urfave/cli/app.go:481 +0x180
github.com/rancher/cli/vendor/github.com/urfave/cli.Command.Run(0x1b6128b, 0x5, 0x0, 0x0, 0x4227bac50, 0x1, 0x1, 0x1b6ea52, 0x19, 0x0, ...)
    /go/src/github.com/rancher/cli/vendor/github.com/urfave/cli/command.go:186 +0x9ed
github.com/rancher/cli/vendor/github.com/urfave/cli.(*App).Run(0x422911800, 0x4228a9d00, 0x9, 0x10, 0x0, 0x0)
    /go/src/github.com/rancher/cli/vendor/github.com/urfave/cli/app.go:235 +0x57d
main.mainErr(0x42295df70, 0x0)
    /go/src/github.com/rancher/cli/main.go:87 +0xc29
main.main()
    /go/src/github.com/rancher/cli/main.go:42 +0x22
Sangeethas-MBP:rancher-v1.0.0-alpha5 sangeethahariharan1$ 
```

**Comment by dramich** [comment]
Fixed [here](https://github.com/rancher/cli/pull/119)

**Comment by deniseschannon** [comment]
Available with v1.0.0-alpha7

**Comment by sangeethah** [comment]
Tested with  v1.0.0-alpha7.
Able to log in with certs having new line characters.
